Makefile

```diff
@@ -7,23 +10,21 @@ GIT_SHA = $(shell git rev-parse --short HEAD)
 GOOS = $(shell go env GOOS)
 ARCH = $(shell go env GOARCH)
 
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
+
 all: generate
 
 version:
 	@echo $(VERSION)
 
-###
-# Setup
-###
-
 clean:
-	rm -rf ./vendor
 	rm -rf ./bin
 
-vendor:
```

Comment:
Probably keep this section, but use...

###
# Setup
###

clean:
        rm -rf ./vendor
        rm -rf ./bin

vendor:
        export GO111MODULE=on; go mod tidy

instead